src/main/webapp/WEB-INF/views/page.jsp

```diff
@@ -0,0 +1,16 @@
+<%--
+  Created by IntelliJ IDEA.
+  User: dalai_s
+  Date: 5/14/2021
+  Time: 7:37 PM
+  To change this template use File | Settings | File Templates.
+--%>
```

Comment:
maybe we should delete this information?

Reply:
agree